modules/network/src/main/java/org/apache/ignite/internal/network/recovery/RecoveryClientHandshakeManager.java

```diff
@@ -165,14 +165,14 @@ public void onInit(ChannelHandlerContext handlerContext) {
     /** {@inheritDoc} */
     @Override
     public void onMessage(NetworkMessage message) {
-        if (message instanceof HandshakeStartMessage) {
```

Comment:
Why did you change the order here?

Reply:
It seems to read better this way (as start and finish handling are now close to each other